packages/desktop-client/src/components/table.tsx

```diff
@@ -819,7 +838,7 @@ export function SelectedItemsButton({ name, items, onSelect }) {
           height={8}
           style={{ marginRight: 5, color: theme.pageText }}
         />
-        {selectedItems.size} {name}
+        {label}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏`Table` component should have no knowledge about transactions/schedules/rules/etc. It should be a generic stand-alone thing. So with that in mind: we should not have special transaction/schedule translations here.

Instead: the consumer component should pass `name={t('transactions')}`.

Reply:
This comes at the cost of having no pluralization knowledge, as the count is only known in this component. As this is how it has been until now, this is an agreeable compromise?

Reply:
For now: I think that's acceptable.

Best case scenario - we could do a bit of refactoring here and pass in `name={(count: number) => t('{count} pluralized')}`.